sdk/translation/ai-translation-document-rest/review/ai-translation-document.api.md

```diff
@@ -59,17 +55,9 @@ export interface CancelTranslationDefaultResponse extends HttpResponse {
 export type CancelTranslationParameters = RequestParameters;
 
 // @public
-function createClient(endpointParam: string, credentials: TokenCredential | KeyCredential, options?: ClientOptions): DocumentTranslationClient;
+function createClient(endpointParam: string, credentials: TokenCredential | KeyCredential, { apiVersion, ...options }?: DocumentTranslationClientOptions): DocumentTranslationClient;
```

Comment:
Why is the `apiVersion` destructured here versus just using the `DocumentTranslationClientOptions`.

Reply:
This is an emitter change.

@joheredi @qiaozha is this related to the RLC API version policy changes? Maybe we shouldn't destructure in this way if it's going to appear in the APIView and the type definitions.

Reply:
yeah, it's related to RLC api version policy change, I think we can take the same change as you have done [in your remove RLC code from Modular PR](https://github.com/Azure/autorest.typescript/pull/2728) if the current change doesn't looks good and @joheredi agrees to it.